src/routes/web.php

```diff
@@ -14,5 +15,8 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('register');
```

Comment:
Better if you move this to the appropriate controller. Keep your route clean, and let your controllers handle your logics.